spec/spec_helper.rb

```diff
@@ -0,0 +1,6 @@
+require 'bundler'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.